usaspending_api/awards/models/financial_accounts_by_awards.py

```diff
@@ -17,6 +17,8 @@ class AbstractFinancialAccountsByAwards(DataSourceTrackedModel):
     parent_award_id = models.TextField(blank=True, null=True)
     fain = models.TextField(blank=True, null=True)
     uri = models.TextField(blank=True, null=True)
+    prior_year_adjustment = models.TextField(blank=True, null=True)
+    pa_reporting_key = models.TextField(blank=True, null=True)
```

Comment:
Do we expect a certain set of values for these fields (e.g. can we use [`choices`](https://docs.djangoproject.com/en/5.1/ref/models/fields/#choices))?

Reply:
Choices is a good idea when we're accepting something from a source that hasn't been validated (maybe user submission). In this case, we're reading data from the broker, which has already performed it's own set of validations. 

We could still potentially add options here, but if anything changes on the schema side (a new PARK is added, we'd need to make sure to update this before hand, or else the submission loader would fail). 

I'd also suspect that in this case, there might be too many options to include here. We have quite a few program activities. 

I can add `help_text` as Seth suggested though.